File: problems/problems_2833/solution.ts
function furthestDistanceFromOrigin(moves: string): number {
    
};

export function Solve(inputJsonElement: string): any {
	const inputValues: string[] = inputJsonElement.split("\n");
	const moves: string = JSON.parse(inputValues[0]);
	return furthestDistanceFromOrigin(moves);
}

File: problems/problems_2833/solution.go
package problem2833

import (
	"encoding/json"
	"log"
	"strings"
)

func furthestDistanceFromOrigin(moves string) int {
    
}

func Solve(inputJsonValues string) any {
	inputValues := strings.Split(inputJsonValues, "\n")
	var moves string

	if err := json.Unmarshal([]byte(inputValues[0]), &moves); err != nil {
		log.Fatal(err)
	}

	return furthestDistanceFromOrigin(moves)
}
